app/src/main/java/com/lucascabral/marvelsuperheroes/helper/ConstantKeys.kt

```diff
@@ -0,0 +1,4 @@
+package com.lucascabral.marvelsuperheroes.helper
+
+const val API_KEY: String = "426a8a3c32e2543316228982d5e3d552"
+const val PRIVATE_KEY: String = "3d286fee9404e7f0c565a623ccd54871b179a2b6"
```

Comment:
Estas chaves poderiam estar no BuildConfig, mais ou menos como te mostrei naquele dia, dos [Build Config Fields](https://developer.android.com/studio/build/gradle-tips#share-custom-fields-and-resource-values-with-your-app-code). Recomendo porque, imaginando um caso mais real de uso de chaves de API, provavelmente não teríamos ela gravada no código; mas, se tivéssemos, seria mais interessante manter essa chave oculta dentro do ambiente de deploy, pra não precisar manter ela no código. Aí, no caso, entra naquele esquema de termos dois ou mais build types (`debug` e `release`), que não é exatamente o nosso caso aqui, mas fica a recomendação.

Ficariam até com o mesmo nome, só seriam acessadas por `BuildConfig.API_KEY` em vez de direto com `API_KEY`. Se não entender totalmente, tranquilo, depois podemos falar mais sobre.

Reply:
Feito.